src/System.Net.Security/src/System/Net/ContextFlagsPal.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace System.Net
+{
+    [Flags]
+    internal enum ContextFlagsPal
```

Comment:
The values are Windows specific as the values are part of the SDK.
My suggestion in these cases is to create an adapter class that will transform from one to another (e.g. from `System.Net.ContextFlagsPal` to `Interop.SSPI.ContextFlags`).

The initial port is still pending a few changes. All these values will need to be renamed to either `ASC_REQ_DELEGATE` or `ISC_REQ_DELEGATE` in the final form to respect the Interop name convention.